runelite-client/src/main/java/net/runelite/client/plugins/metronome/MetronomePlugin.java

```diff
@@ -69,14 +68,25 @@ public void onGameTick(GameTick tick)
 
 		if (++tickCounter % config.tickCount() == 0)
 		{
+			// As playSoundEffect only uses the provided volume when in-game sound effects are muted, the sound effect
+			// volume needs to be set to the desired volume then reset after to what it was initially configured to.
+			int previousVolume = client.getSoundEffectVolume();
+
 			if (config.enableTock() && shouldTock)
 			{
-				client.playSoundEffect(SoundEffectID.GE_DECREMENT_PLOP, SoundEffectVolume.MEDIUM_HIGH);
+				int tockVolume = config.tockVolume().getVolume();
+				client.setSoundEffectVolume(tockVolume);
```

Comment:
Why is `setSoundEffectVolume` used along with `playSoundEffect(int id, int volume)`? The documentation for the latter method explicitly states:

> Plays a sound effect, even if the player's sound effect volume is muted.